tutorials/intro/40_sensor_locations.py

```diff
@@ -58,38 +55,36 @@
 ###############################################################################
 # .. sidebar:: Computing sensor locations
 #
-#     If you are interested in how standard ("idealized") EEG sensor positions
-#     are computed on a spherical head model, the `eeg_positions`_ repository
-#     provides code and documentation to this end.
+#     If you are interested in how standard (idealized) EEG sensor positions
+#     are computed on a spherical head model, make sure to check out the
+#     `eeg_positions`_ repository.
 #
-# These built-in EEG montages can be loaded via
-# :func:`mne.channels.make_standard_montage`. Note that when loading via
-# :func:`~mne.channels.make_standard_montage`, provide the filename *without*
-# its file extension:
+# These built-in EEG montages can be loaded with
+# :func:`mne.channels.make_standard_montage` (note that you need to provide the
+# filename *without* its extension):
 
 ten_twenty_montage = mne.channels.make_standard_montage('standard_1020')
 print(ten_twenty_montage)
 
 ###############################################################################
-# Once loaded, a montage can be applied to data via one of the instance methods
-# such as :meth:`raw.set_montage <mne.io.Raw.set_montage>`. It is also possible
-# to skip the loading step by passing the filename string directly to the
-# :meth:`~mne.io.Raw.set_montage` method. This won't work with our sample
-# data, because it's channel names don't match the channel names in the
-# standard 10-20 montage, so these commands are not run here:
+# Once loaded, a montage can be applied to data with the
+# :meth:`~mne.io.Raw.set_montage` method, for example
+# :meth:`raw.set_montage <mne.io.Raw.set_montage>`. It is also possible to skip
+# the loading step by passing the filename string directly to the
+# :meth:`~mne.io.Raw.set_montage` method. This will not work with our sample
+# data, because its channel names do not match the channel names in the
+# standard 10-20 montage. Therefore, we do not run the following commands here:
 
 # these will be equivalent:
 # raw_1020 = raw.copy().set_montage(ten_twenty_montage)
 # raw_1020 = raw.copy().set_montage('standard_1020')
 
 ###############################################################################
 # :class:`Montage <mne.channels.DigMontage>` objects have a
-# :meth:`~mne.channels.DigMontage.plot` method for visualization of the sensor
-# locations in 3D; 2D projections are also possible by passing
-# ``kind='topomap'``:
+# :meth:`~mne.channels.DigMontage.plot` method for visualizing the sensor
+# locations in 2D or 3D:
 
-fig = ten_twenty_montage.plot(kind='3d')
-fig.gca().view_init(azim=70, elev=15)
+ten_twenty_montage.plot(kind='3d').gca().view_init(azim=70, elev=15)
```

Comment:
-1 on this change.  the method chaining is not necessary for the basic plot to work, and this is IMO harder to read and it's now harder to copy/paste just the crucial part.  I'd rather keep the lines separate and add a comment to the second line like `# set the view angle` or similar.